infra/base-images/base-builder/oss_fuzz_on_demand_build

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash -eux
```

Comment:
This is named differently from `build_oss_fuzz_on_demand` which are referenced in the Dockerfile and compile scripts. Is this correct?